modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -39,27 +47,68 @@ EOF
 The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.15.0` to download the {microshift-short} 4.15.0 RPMs.
 ====
 
+. Optional. Use the blueprint installed in the `/usr/share/microshift/blueprint` directory that is specific to your platform architecture. See the following example snippet for an explanation of the blueprint sections:
++
+.Generated Image Builder blueprint example snippet
++
+[source,text]
+----
+name = "microshift-blueprint"
+description = "MicroShift 4.15.1 on x86_64 platform"
+version = "0.0.1"
+modules = []
+groups = []
+
+[[packages]] <1>
+name = "microshift"
+version = "4.15.1"
+...
+...
+
+[customizations.services] <2>
+enabled = ["microshift"]
+
+[customizations.firewall]
+ports = ["22:tcp", "80:tcp", "443:tcp", "5353:udp", "6443:tcp", "30000-32767:tcp", "30000-32767:udp"]
+...
+...
+
+[[containers]] <3>
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:f41e79c17e8b41f1b0a5a32c3e2dd7cd15b8274554d3f1ba12b2598a347475f4"
+
+[[containers]]
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:dbc65f1fba7d92b36cf7514cd130fe83a9bd211005ddb23a8dc479e0eea645fd"
+...
+…
+EOF
+----
+<1> References for all non-optional {microshift-short} RPM packages using the same version compatible with the `microshift-release-info` RPM.
+<2> References for automatically enabling {microshift-short} on system startup and applying default networking settings.
+<3> References for all non-optional {microshift-short} container images necessary for an offline deployment.
+
 . Add the blueprint to the Image Builder by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints push minimal-microshift.toml
+$ sudo composer-cli blueprints push <microshift-blueprint.toml> <1>
 ----
+<1> Replace <microshift-blueprint.toml> with the name of your TOML file.
 
 .Verification
 
 . Verify the Image Builder configuration listing only {microshift-short} packages by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints depsolve minimal-microshift | grep microshift
+$ sudo composer-cli blueprints depsolve <microshift-blueprint> | grep microshift <1>
 ----
+<1> Replace <microshift-blueprint> with the name of your TOML file.
 +
 .Example output
 +
 [source,terminal]
 ----
-blueprint: minimal-microshift v0.0.1
+blueprint: microshift-blueprint v0.0.1
```

Comment:
I know that the example is potentially outside the scope of the review, but shouldn't this reflect the version we are targetting? 4.13 seems out of place here.

Reply:
Right, we can replace `4.13.1` by `4.15.1` at least.
I'm not sure what to do with the dates and commits though, i.e. `202305250827.p0.g4105d3`.

Maybe, we should remove this output all together?

Reply:
So Should I remove this output file?